jujugui/static/gui/src/app/components/machine-view/machine/_machine.scss

```diff
@@ -95,6 +89,14 @@
         margin-right: 10px;
         padding: 2px;
         border: 1px solid $canvas-background;
-        border-radius: 6px;
+        border-radius: 50%;
+
+        img {
```

Comment:
We shouldn't target the element here. As a component we shouldn't only target classes as the element may change but we don't want the styling to be lost. For example this element maybe be changed to picture in the future which with require two edits.